### PR TITLE
Added msft-windows-disable-copilot-and_recall.ps1

### PR DESCRIPTION
**Pull request by coop-a-loop**
Added msft-windows-disable-copilot-and_recall.ps1, which turns off Copilot, hides the Copilot taskbar icon and disables Recall

### CODE REVIEW

File: msft-windows/msft-windows-disable-copilot-and-recall.ps1 [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+$loggedOnUserName = $loggedOnUser.username
+$UserWithoutDomain = $loggedOnUserName -replace "^.*?\\"
+$GetSID = Get-ChildItem -Path $profileList -rec -ea SilentlyContinue | % { if((get-itemproperty -Path $_.PsPath) -match "$userWithoutDomain") { $_.PsPath} }
+$SID = $GetSID -replace "^.*?list\\"
+Write-Host "Current Logged On User: $loggedOnUserName `n"
+
+# Add HKEY_USERS drive so HKU can be referenced reg path variables
+Write-Host "Creating drive mapped to HKEY_USERS"
+New-PSDrive -PSProvider Registry -Name HKU -Root HKEY_USERS
+Write-Host "Drive mapped to HKEY_USERS `n"
```
Comment (nettts) [inline msft-windows/msft-windows-disable-copilot-and-recall.ps1:66]: Can you remove this unix 'n? If you have this problem in the future, notepad++ or visual studio code.

File: msft-windows/msft-windows-disable-copilot-and-recall.ps1 [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    
+}
+
+# Start the script logic here. This is the part that actually gets done what you need done.
+
+Start-Transcript -Path $LogPath
+
+Write-Host "Description: $Description"
+Write-Host "Log path: $LogPath"
+Write-Host "RMM: $RMM `n"
```
Comment (nettts) [inline msft-windows/msft-windows-disable-copilot-and-recall.ps1:48]: Remove "`n"

File: msft-windows/msft-windows-disable-copilot-and-recall.ps1 [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+# Variables need set in RMM
+
+# Get SID for currently logged on user
+$profileList = "HKLM:SOFTWARE\Microsoft\Windows NT\CurrentVersion\ProfileList"
+$loggedOnUser = Get-Ciminstance -ClassName Win32_ComputerSystem | Select-Object UserName
+$loggedOnUserName = $loggedOnUser.username
+$UserWithoutDomain = $loggedOnUserName -replace "^.*?\\"
+$GetSID = Get-ChildItem -Path $profileList -rec -ea SilentlyContinue | % { if((get-itemproperty -Path $_.PsPath) -match "$userWithoutDomain") { $_.PsPath} }
+$SID = $GetSID -replace "^.*?list\\"
+Write-Host "Current Logged On User: $loggedOnUserName `n"
```
Comment (nettts) [inline msft-windows/msft-windows-disable-copilot-and-recall.ps1:61]: Remove "`n"

**Review by nettts — CHANGES_REQUESTED** [review]
There is some "`n" end of lines that need removed. I'm sure this is because of the text or code editor you were using before really getting into the code.

Or a copy paste from other sources. Doesn't matter. Just needs removed :).

**Comment by coop-a-loop** [comment]
I wrote my code in PowerShell ISE and used 'n to insert a new line to
better format my output with the write-host command. Let me know if you
still want it removed.

https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_special_characters?view=powershell-7.4

**Pull request merged**